.circleci/config.yml

```diff
@@ -2,8 +2,8 @@
 # syntax, such as anchors, will be fixed automatically.
 version: 2.1
 orbs:
-  shared: getoutreach/shared@2.19.2
-  queue: eddiewebb/queue@1.8.4
+  shared: getoutreach/shared@dev:first
```

Comment:
This seems not quite right and I can't recall what it should be instead.

Reply:
Yeah it seemed odd to me too... was having issues with stencilbot so ran stencil manually on this

Reply:
This is because devbase is using unstable. Because of the issues with the dev CircleCI orb (it expires after some time), we may want to use devbase RC / stable instead.